ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts

```diff
@@ -45,8 +47,43 @@ export default {
       ...fontVariants.sansRegular,
       ...fontSizes.pica,
     }),
+  checkboxContainer: () =>
+    css({
+      display: 'flex',
+      flexWrap: 'nowrap',
+    }),
+
   textArea: () =>
     css({
       resize: 'none',
     }),
+  checkboxLabel: ({ spacings }: Theme) =>
+    css({
+      flex: 'auto',
+      marginLeft: `${spacings.DOUBLE}rem`,
```

Comment:
We'll need `marginInlineStart` to allow for RTL languages to have the correct spacing